DocumentService/Repositories/DocumentRepository.cs

```diff
@@ -142,6 +162,57 @@ private async Task<DocumentInfo> GetDocument(Guid id)
             return await query.FirstOrDefaultAsync();
 
         }
+        private DateTime GetTimeForNCR()
+        {
+            try
+            {
+                DateTime timeUTC = DateTime.UtcNow;
+                TimeZoneInfo estZone = TimeZoneInfo.FindSystemTimeZoneById("Eastern Standard Time");
+                DateTime estTime = TimeZoneInfo.ConvertTimeFromUtc(timeUTC, estZone);
+                return estTime;
+            }
+            catch (TimeZoneNotFoundException)
+            {
+                throw;
+            }
+            
+        }
```

Comment:
We want to use DateTime.UtcNow and not convert it to easter standard time in the database so we want to remove this.
```suggestion
```